app/src/utils/transfer.ts

```diff
@@ -25,12 +25,13 @@ export const convertAmount = (input: number | null, token: Token | null): bigint
  * @param token - The token object which includes its decimals property.
  * @returns The amount readable by humans
  */
-export const toHuman = (input: bigint, token: Token): number => {
+export const toHuman = (input: bigint | string, token: Token): number => {
   return Number(input) / 10 ** token.decimals
 }
```

Comment:
Does that work? Won't we lose precision here?

```suggestion
export const toHuman = (input: bigint | string, token: Token): number => {
  let value: bigint
  if (typeof input === 'bigint') value = input
  else value = BigInt(input)
  
  const factor = BigInt(10 ** token.decimals)
  const result = value / factor

  return Number(result);
}
```

Reply:
it can just be 

```
let value = (typeof input === 'bigint') ? input : BigInt(input)

return Number(value) / 10 ** token.decimals

```

Reply:
Ticket created [here](https://www.notion.so/velocitylabs/df1914bfea20494c8a0a32af79fbb448?v=aadc09f0be594111b29113e58ee73b42&p=0ace274b2d374dd58b78adfe1cf5360f&pm=s)